layermetadata/layer_metadata_restorer.go

```diff
@@ -1,7 +1,8 @@
-package lifecycle
+package layermetadata
```

Comment:
Can this be an internal package?

Reply:
We should also move layer_metadata_restorer_test.go

Reply:
Please see https://github.com/buildpacks/lifecycle/pull/724/commits/35dce2dac769d44a8f367a3558346f95cc19981b.